queries.md

```diff
@@ -2,20 +2,85 @@
 
 ## find all customers that live in London. Returns 6 records
 
+```sql
+
+SELECT CustomerName
+FROM Customers
+WHERE City = 'London'
+
+```
+
 ## find all customers with postal code 1010. Returns 3 customers
 
+```sql
+
+SELECT CustomerName
+FROM Customers
+WHERE PostalCode = '1010'
+
+```
+
 ## find the phone number for the supplier with the id 11. Should be (010) 9984510
 
+```sql
+
+SELECT Phone
+FROM Suppliers
+WHERE SupplierID = 11
+
+```
+
 ## list orders descending by the order date. The order with date 1997-02-12 should be at the top
 
+```sql
+
+SELECT *
+FROM Orders
+ORDER BY OrderDate desc
+
+```
+
 ## find all suppliers who have names longer than 20 characters. You can use `length(SupplierName)` to get the length of the name. Returns 11 records
 
+```sql
+
+SELECT *
+FROM Suppliers
+WHERE LEN(SupplierName) > 20
```

Comment:
syntax, spelling error, i think you have it mixed up with the server function

Reply:
I tried length but it wouldn't work - only LEN would in the editor for me for some reason....

Reply:
LOL of course it works now and LEN doesn't .. odd I'm sure I had it backwards..
Fixed.